docs/src/content/docs/resources/plugins.mdx

```diff
@@ -63,6 +63,12 @@ Extend your site with official plugins supported by the Starlight team and commu
 		title="starlight-image-zoom"
 		description="Add zoom capabilities to your documentation images."
 	/>
+	<LinkCard
+		href="https://github.com/lorenzolewis/starlight-multi-sidebar"
+		title="starlight-multi-sidebar"
+		description="Have multiple sidebars in your Starlight site"
```

Comment:
Small nit on the wording as “have” is a bit passive and other descriptions here tend to be “add”, “create”, “publish” etc.

```suggestion
		description="Enable multiple sidebars in your Starlight site"
```